uaclient/entitlements/tests/test_fips.py

```diff
@@ -105,10 +105,12 @@ def test_messaging_passes_assume_yes(
 
 
 class TestFIPSEntitlementCanEnable:
+    @mock.patch("uaclient.util.is_config_value_true")
```

Comment:
Can add the return_value=False here instead of adding it into the test method below to save a line
```suggestion
    @mock.patch("uaclient.util.is_config_value_true", return_value=False)
```